middle_end/flambda2/parser/print_fexpr.ml

```diff
@@ -255,6 +257,7 @@ let const ppf (c : Fexpr.const) =
   | Naked_int32 i -> Format.fprintf ppf "%lil" i
   | Naked_int64 i -> Format.fprintf ppf "%LiL" i
   | Naked_nativeint i -> Format.fprintf ppf "%Lin" i
+  | Naked_vec128 { high; low } -> Format.fprintf ppf "%Ld:%Ld" high low
```

Comment:
Using `"%016Lx"` rather than `"%Ld"` might give a more consistent output (same for other printing functions later)

Reply:
Changed to `%016Lx`